Tribler/Test/test_tunnel_community.py

```diff
@@ -19,6 +19,7 @@
 #TODO(emilon): Quick hack to get 6.4.1 out the door, (re tunnel_community tests disabling is_unit_testing flag)
 from os import environ
 environ["SKIP_OPTIN_DLG"] = "True"
+environ["TRIBLER_LOAD_DISCOVERY_COMMUNITY"] = "False"
```

Comment:
This will disable the discovery community for all the tests executed after these.
I guess it would be better to add a flag to TestGuiAsServer.setUp()